appeals/api/src/server/config/config.js

```diff
@@ -40,6 +40,9 @@ const { value, error } = schema.validate({
 		template: {
 			validAppellantCase: {
 				id: '3b4b74b4-b604-411b-9c98-5be2c6f3bdfd'
+			},
+			appealConfirmed: {
+				id: '783f94cc-1d6d-4153-8ad7-9070e449a57c'
```

Comment:
Not part of the ticket by the looks of it, but we're going to need to get these from environment variables.